ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -61,8 +61,8 @@ if(COMMAND ie_faster_build)
     )
 endif()
 
-target_link_libraries(onnx_importer PRIVATE onnx_common ngraph::builder onnx onnx_proto ${Protobuf_LIBRARIES}
```

Comment:
As I understand `${Protobuf_LIBRARIES}` is needed if `NGRAPH_USE_SYSTEM_PROTOBUF` is `ON`, but maybe no one uses such configuration and we can remove it?